src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue

```diff
@@ -248,14 +267,12 @@ export const config = [
 }
 
 .seventv-hide-player-ext {
```

Comment:
Instead here, rather instead of disabling stylelint rules, just use the encapsulation formatting

```suggestion
.seventv-hide-player-ext {
    .video-player {
      .extension-taskbar, .extension-container,
	  .extensions-dock__layout, .extensions-notifications,
	  .extensions-video-overlay-size-container {
		display: none !important;
	  }
	}
}
```